noisyatom/blogs/tests/test_models.py

```diff
@@ -0,0 +1,56 @@
+
+from django.test import TestCase
+from django.utils.text import slugify
+
+from blogs.models import Post
+
+
+class PostModelTest(TestCase):
+
+	def setUp(self):
+		self.post = Post.objects.create(title='New title test', slug='first-new-title-test-created')
+
+	def create_post(self, title='Post new blog'):
+		return Post.objects.create(title=title)
+
+	def test_create_title(self):
```

Comment:
This does not create a title? It looks like it is testing the string matches a post with the defined slug title. Maybe write a comment on the method?